subsurface-mobile-main.cpp

```diff
@@ -88,10 +88,9 @@ int main(int argc, char **argv)
 
 void set_non_bt_addresses()
 {
-#if SERIAL_FTDI
-	connectionListModel.addAddress("FTDI");
-#endif
 #if defined(Q_OS_ANDROID)
+    connectionListModel.addAddress("usb-serial-for-android");
```

Comment:
and of course then here `"usb-serial"` as well

Reply:
fixed